pymc/distributions/multivariate.py

```diff
@@ -224,6 +224,9 @@ def dist(cls, mu, cov=None, tau=None, chol=None, lower=True, **kwargs):
         cov = quaddist_matrix(cov, chol, tau, lower)
         return super().dist([mu, cov], **kwargs)
 
+    def get_moment(rv, size, mu, cov):
```

Comment:
You will need special logic for when size is not None, and possibly when the shape of `cov` broadcasts `mu`

Reply:
`(np.ones(2), np.eye(2), (4, 2), np.ones((4, 2, 2))) `